site/public/css/polls.css

```diff
@@ -64,6 +75,11 @@ input[type="radio"][disabled=""]:checked::before {
     background: var(--standard-medium-dark-gray);
 }
 
+input[type="checkbox"][name="answers[]"] {
+    transform: scale(0.75);
+    margin: 0 1px 0 0;
+}
```

Comment:
Is there a better way to target this?  In general, it's better to target things with classes or IDs.

Reply:
This should be good enough. I could add classes for the radio buttons and checkboxes that specifically in the answer field but I thought that wasn't necessary because there's already a name defined.

Reply:
Yeah, I eventually did some class manipulation to make the targeting in the new table easier.